src/test/codegen/intrinsics/mask.rs

```diff
@@ -0,0 +1,11 @@
+#![crate_type = "lib"]
+#![feature(core_intrinsics)]
+
+// CHECK-LABEL: @mask_ptr
+// CHECK-SAME: [[WORD:i[0-9]+]] %mask
+#[no_mangle]
+pub fn mask_ptr(ptr: *const u16, mask: usize) -> *const u16 {
+    // CHECK: call
+    // CHECK-SAME: @llvm.ptrmask.{{p0|p0i8}}.[[WORD]](ptr %ptr, [[WORD]] %mask)
```

Comment:
I'm unsure: Should this maybe be something like
```suggestion
    // CHECK-SAME: @llvm.ptrmask.{{p0|p0i8}}.[[WORD]]({{ptr|i8\*}} %ptr, [[WORD]] %mask)
```
to make it not fail on older LLVM that's (presumably) not using opaque pointers?

Reply:
Right, yep. I was confused by types in types/types in function names being different

Reply:
Should be good now